src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -156,13 +156,18 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
 #endif
             app.UseStaticFiles();
+
+            app.UseEndpointRouting(routes =>
+            {
+                routes.MapApplication();
```

Comment:
Change to MapAssembly for web apis
* Controllers - controller type
* Razor pages - application parts - follow the assembly for embedded pages

Reply:
I thought we wanted MapAssembly for everything? Are there bugs blocking this?